CMakeLists.txt

```diff
@@ -602,6 +609,10 @@ if (ENABLE_BUILD_FOR_PPA)
    endif ()
    set (PKG_CONFIG_LIBS "${PKG_CONFIG_LIBS} -pthread")
 endif ()
+if (MONGOCRYPT_DFP_DIR STREQUAL "USE-SYSTEM")
+   get_property (SYSTEM_DFP_LOC TARGET intel_dfp PROPERTY IMPORTED_LOCATION)
+   set (PKG_CONFIG_LIBS "${PKG_CONFIG_LIBS} ${SYSTEM_DFP_LOC}")
```

Comment:
Is there not a pkg-config for DFP that one can depend on transitively?

Reply:
Sadly, there is not one.  This is basically "here's a box of Lego, see what you can make from it"